### PR TITLE
Corrected 4 spelling errors in arm9/video.h preamble.

### PR DESCRIPTION
**Pull request by awiebe**
Just spell checking

### CODE REVIEW

File: include/nds/arm9/video.h [hunk 1]
```diff
@@ -37,10 +37,10 @@
 	uses. They can hold the graphics for your sprites, the textures for your 3D
 	space ships, the tiles for your 2D platformer, or a direct map of pixels to
 	render to the screen. Figuring out how to effectively utilize this flexible but
-	limited amount of memory will be one the most challenging endeavors you will
```
Comment (mtheall) [inline include/nds/arm9/video.h]: Both of these are a correct spelling, but I think we tend to stick with the common forms found in U.S. English, in this case "endeavor".

File: include/nds/arm9/video.h [hunk 2]
```diff
@@ -54,8 +54,8 @@
 	tile and map offsets will be lost.
 
 	VRAM banks can be mapped to specific addresses for specific purposes.  In our
-	case, any of the 4 main banks amd several of the smaller ones can be mapped to
-	the main 2D background engine.(A B C and D banks are refered to as ìmainî
```
Comment (mtheall) [inline include/nds/arm9/video.h]: I would probably change `ìmainî` to `"main"`.

Comment (awiebe) [inline include/nds/arm9/video.h]: We're sure that's not like an identifier or anything?

Comment (awiebe) [inline include/nds/arm9/video.h]: Perhaps it could say
"are collectively referred to as main VRAM"

Comment (awiebe) [inline include/nds/arm9/video.h]: Oh yea, just noticed the strange accents, just some garbled text, definitely not an identifier, will modify.

Comment (mtheall) [inline include/nds/arm9/video.h]: If I had to guess, they probably got copied from the left/right single/double quotes ‘’/“” from a different character encoding.

**Pull request merged**